zfsbootmenu/lib/zfsbootmenu-lib.sh

```diff
@@ -78,17 +78,22 @@ draw_be() {
 
   zdebug "using environment file: ${env}"
 
+  if [ -f "${BASE}/cmdline" ]; then
+    kcl_text="[CTRL+T] revert kcl"
+    kcl_bind=",alt-t"
+  fi
+
   header="$( column_wrap "\
 [RETURN] boot:[ESCAPE] refresh view:[CTRL+P] pool status
 [CTRL+D] set bootfs:[CTRL+S] snapshots:[CTRL+K] kernels
 [CTRL+E] edit kcl:[CTRL+J] jump into chroot:[CTRL+R] recovery shell
-[CTRL+L] view logs::[CTRL+H] help" \
+[CTRL+L] view logs:${kcl_text}:[CTRL+H] help" \
 "\
 [RETURN] boot
 [CTRL+R] recovery shell
 [CTRL+H] help" )"
 
-  expects="--expect=alt-e,alt-k,alt-d,alt-s,alt-c,alt-r,alt-p,alt-w,alt-j,alt-o"
+  expects="--expect=alt-e,alt-k,alt-d,alt-s,alt-c,alt-r,alt-p,alt-w,alt-j,alt-o${kcl_bind}"
```

Comment:
```suggestion
  expects="--expect=alt-e,alt-k,alt-d,alt-s,alt-c,alt-r,alt-p,alt-w,alt-j,alt-o${kcl_bind:+,${kcl_bind}}"
```
and using
```sh
kcl_bind="alt-t"
```
would be more verbose, but maybe a bit more obvious in intent.

Reply:
Yeah, that's a bit more obvious. Done!